### PR TITLE
Initial samples

### PR DESCRIPTION
**Pull request by sameerajayasoma**
Adding the initial set of samples

### CODE REVIEW

File: gcalendar-new-event-to-trello-card/main.bal [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+// Google Calendar configuration parameters
+configurable calendar:ListenerConfig config = ?;
+
+// Trello configuration parameters
+configurable string trelloApiKey = ?;
+configurable string trelloApiToken = ?;
+configurable string trelloListId = ?;
+
+listener calendar:Listener calendarListener = new (config, httpListener);
```
Comment (anupama-pathirage) [inline gcalendar-new-event-to-trello-card/main.bal]: Shall we move the Calendar listener related configs above near to the service.

File: gcalendar-new-event-to-trello-card/main.bal [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    }
+}
+
+function transform(calendar:Event calEvent) returns trello:Cards => {
+    name: calEvent.summary,
+    due: calEvent.end?.dateTime,
+    idList: trelloListId,
+    desc: string `New event is created on Google Calendar: ${calEvent.summary ?: ""}. 
+        The event starts on ${calEvent.'start?.dateTime ?: ""} and ends on ${calEvent.end?.dateTime ?: ""}`
+};
```
Comment (anupama-pathirage) [inline gcalendar-new-event-to-trello-card/main.bal]: Missing new line

**Comment by anupama-pathirage** [comment]
Shall we add a sample Config.toml also or add a readme with those? Otherwise, some config formats are hard to derive, e.g. `calendar:ListenerConfig`

File: gcalendar-new-event-to-trello-card/main.bal (hunk 1 above)
Comment (sameerajayasoma) [reply gcalendar-new-event-to-trello-card/main.bal]: Fixed with 686f1420fcca26c7b3fc489f6cb12737010c5a0f

File: gcalendar-new-event-to-trello-card/main.bal (hunk 2 above)
Comment (sameerajayasoma) [reply gcalendar-new-event-to-trello-card/main.bal]: Done.

**Comment by sameerajayasoma** [comment]
> 

I added README.md files for three complicated samples. I will add README.md files for the remaining samples soon. Please review and merge if there are no other issues.

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**